Cargo.lock

```diff
@@ -265,6 +257,30 @@ dependencies = [
  "either",
 ]
 
+[[package]]
+name = "jiff"
```

Comment:
Complained about all this stuff:

https://github.com/rust-cli/env_logger/pull/352#issuecomment-2747948788

Seems like there is potential for it to be reverted.

Reply:
Ah, looks like they're lockfile false positives due to a Cargo bug.